examples/react-vanilla-ts-src/vite.config.ts

```diff
@@ -0,0 +1,14 @@
+import { join } from 'node:path'
+import viteFastify from '@fastify/vite/plugin'
+import viteReact from '@vitejs/plugin-react'
+
+export default {
+  root: join(import.meta.dirname, 'src', 'client'),
+  plugins: [
+    viteFastify(), 
+    viteReact()
+  ],
+  build: {
+    outDir: join(import.meta.dirname, 'build'),
```

Comment:
Did you want to add a comment here saying it must match the server.ts `distDir` folder?